woocommerce/wc-admin
# Refactor CustomerEffortScore as Package

Building from PR https://github.com/woocommerce/woocommerce-admin/pull/5324, this extracts the `CustomerEffortScore` into its own package.

This required more effort than I was expecting. I followed the [instructions to create a package](https://github.com/woocommerce/woocommerce-admin/pull/5324) but additional steps were required in the webpack build and WP script registration. I would like to update the documentation there once this PR lands.

This is not yet published on the NPM repo.

Some code formatting changes  occurred automatically on the moved files upon commit.

### Detailed test instructions:

- Build the package:

```
$ npm install
$ npm run start
```

- Add new product and confirm "Provide effort for adding a new product" modal appears.
